subsys/usb/device/class/Kconfig.bt

```diff
@@ -1,18 +1,9 @@
 # Copyright (c) 2016 Wind River Systems, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-menuconfig USB_DEVICE_BLUETOOTH
+config USB_DEVICE_BLUETOOTH
 	bool "USB Bluetooth Device Class support"
 	select BT
 	select BT_HCI_RAW
 	help
 	  USB Bluetooth device class support
-
-config USB_DEVICE_BLUETOOTH_VS_H4
-	bool "USB Bluetooth H4 vendor command"
-	depends on USB_DEVICE_BLUETOOTH
-	select BT_HCI_RAW_H4
-	select BT_HCI_RAW_CMD_EXT
-	help
-	  Enables vendor command to switch to H:4 transport using the bulk
-	  endpoint.
```

Comment:
Doesn't this need to be deprecated first?

Reply:
> Doesn't this need to be deprecated first?

I definitely considered it, but IMO we'd create unreasonable overhead for us with it. See the bullet points I listed in the PR description.

Reply:
Yeah I noticed that you wrote 

> The PR also removes the experimental H:4 encoding support from the USB transport for several reasons:

But it's not marked as experimental and has existed for > 4 years. Maybe the maintainer of USB (@jfischer-no) has some input here. 

If it is truly experimental, then it's fine to remove IMO (although we should possibly add that to the release notes and/or migration guide?)

Reply:
The experiment is still ongoing. But "USB Bluetooth H4 vendor command" is not needed (the experiment only uses BT_HCI_RAW_H4 functionality) and can go away.